upgrade/util/util.go

```diff
@@ -144,3 +148,732 @@ func CreateOrUpdateLonghornVersionSetting(namespace string, lhClient *lhclientse
 	}
 	return nil
 }
+
+func ListSettings(namespace string, lhClient *lhclientset.Clientset, resourceMaps map[string]interface{}) (map[string]*v1beta2.Setting, error) {
```

Comment:
Can we change to the function name to below or any similar? The same change should be applied to other functions. 

```
// ListAndUpdateSettingsInProvidedCache list all settings and save them into the provided cached `resourceMaps`. This method is not thread-safe.
func ListAndUpdateSettingsInProvidedCache(...) {
    ...
```